data/projects/ronin_dojo.mdx

```diff
@@ -15,4 +15,4 @@ tags: ['Bitcoin', 'Privacy']
 
 ## About this project
 
-RoninDojo develops hardware and software to make it easy for anyone to use bitcoin privately as if it were cash. Some of our products include bitcoin nodes, steel backup plates, powerful software, premium support, and a lot more coming soon. We are working hard to increase accessibility for bitcoin users to connect their wallet to a node, use privacy enhancing coinjoin, do their own self custody, and so much more. Financial sovereignty is our goal and we won't slow down until it is achieved!
+RoninDojo products include Bitcoin nodes, steel backup plates, powerful storage, premium support, and much more coming soon. RoninDojo works hard to increase accessibility for Bitcoiners to connect their wallet to a node, use privacy-enhancing CoinJoin, and do self-custody.
```

Comment:
I don't think we should focus on the product side of this project, but on the fact that this is an open-source operating system that can run on many different devices: https://code.samourai.io/ronindojo/RoninOS